exercism/annalyns-infiltration/LincolnInitialSolution.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+
+static class QuestLogic
+{
+    public static bool CanFastAttack(bool knightIsAwake)
+    {
+        if(knightIsAwake){
```

Comment:
I was going to say it seems to be idiomatic C# to do a newline before braces (that's different than, e.g., TypeScript).

There's an idea in programming called "[bike shedding](https://en.wikipedia.org/wiki/Law_of_triviality)," which is the example

> of a fictional committee whose job was to approve the plans for a nuclear power plant spending the majority of its time on discussions about relatively minor but easy-to-grasp issues, such as what materials to use for the staff bicycle shed, while neglecting the proposed design of the plant itself, which is far more important and a far more difficult and complex task.

The usual solution to bike-shedding code style is to adopt a formatter like [this](https://csharpier.com/) one. It has opinionated defaults and can be configured to run on save or when you trigger it. It helps you just not have to think about code formatting.